config/crd/bases/_.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
This file gets generated for me as well for some time, unsure where the source for this is. It should not be committed (and the generation fixed as well)